src/library/scala/collection/Iterator.scala

```diff
@@ -1,6 +1,7 @@
 package scala.collection
 
 import java.io.{ObjectInputStream, ObjectOutputStream}
+import java.util.Objects
```

Comment:
The method you're using is so trivial that I don't think it's worth adding this dependency.